changelog.txt

```diff
@@ -0,0 +1,2 @@
+- important - [breaking change] The Firebase Functions SDK no longer supports Node 6. Developers must use Node 8.13.0 or higher.
+- important - [breaking change] The Firebase Functions SDK no longer supports Firebase Tools SDK below major version 5.
```

Comment:
Can you add another line here to highlight the env var deprecation? Something like this:

`- important - [breaking change] FIREBASE_PROJECT environment variable is no longer supported and has been supplanted by FIREBASE_CONFIG.`

Reply:
Added in 682e8c8f05619e8c0a8c9dbbce80f7699111c19d